samza-test/src/test/java/org/apache/samza/processor/TestZkStreamProcessorBase.java

```diff
@@ -93,6 +98,41 @@ public void setUp() {
     createTopics(inputTopic, outputTopic);
   }
 
+  protected Thread expireSession1(ZkClient zkClient) {
```

Comment:
This looks like dead code that can be removed. Suspend might be problematic if the code is suspended while invoking a callback (see Thread#suspend docs for details).